.travis.yml

```diff
@@ -31,11 +31,14 @@ script:
       (
         set -x;
         cargo fmt -- --write-mode=diff &&
-        cargo build &&
-        cargo test  --verbose
+        cargo rustc --verbose -- -Zno-trans &&
```

Comment:
Why do we want to do a debug build first and then re-do the release build and test it again below ? Why not just run the one below `cargo test --release --verbose` ?